integration/config.sh

```diff
@@ -41,7 +41,7 @@ has() {
     local host=$2
     local name=$3
     local count=${4:-1}
-    assert "curl -s http://${host}:4040/api/topology/${view}?system=show | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
+    assert "curl -s http://${host}:4040/api/topology/${view}? | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
 }
```

Comment:
Hm. Hard to say without carefully reading the whole thing, but I suspect the desired behaviour here is to "show all". So you should pass in the value corresponding to that. If that's the default, what you have here is fine (though nit: if you have no params, just omit the `?` entirely)